projects/create-and-vary-a-licence-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/provider/entity/Provider.kt

```diff
@@ -108,3 +110,47 @@ interface BoroughRepository : JpaRepository<Borough?, Long?> {
     )
     fun findActiveByCode(code: String): Borough?
 }
+
+@Immutable
+@Entity
+@Table(name = "office_location")
+class OfficeLocation(
+
+    @Column(name = "code", columnDefinition = "char(7)")
+    val code: String,
+
+    val description: String,
+    val buildingName: String?,
+    val buildingNumber: String?,
+    val streetName: String?,
+
+    @Column(name = "district")
+    val districtStr: String?,
+
+    val townCity: String?,
+    val county: String?,
+    val postcode: String?,
+    val telephoneNumber: String?,
+    val startDate: LocalDate,
+    val endDate: LocalDate?,
+
+    @JoinColumn(name = "district_id")
+    @ManyToOne
+    val district: District,
+
+    @Id
+    @Column(name = "office_location_id")
+    val id: Long
+)
+
+interface OfficeLocationRepository : JpaRepository<OfficeLocation?, Long?> {
```

Comment:
Should these types be nullable here?  Not sure whether it actually makes a difference

Reply:
yeah no reason for these being null. Not sure why i did that. Ive changed. Thanks

Reply:
ive also changed the other instance in that class